osslsigncode.c

```diff
@@ -3037,9 +3086,53 @@ static void pe_extract_page_hash(SpcAttributeTypeAndOptionalValue *obj,
 }
 
 static int pe_verify_pkcs7(PKCS7 *p7, char *indata, FILE_HEADER *header,
-			int allownest, GLOBAL_OPTIONS *options)
+		GLOBAL_OPTIONS *options);
+
+static int pe_verify_nested_signature(PKCS7 *p7, char *indata, FILE_HEADER *header,
+		GLOBAL_OPTIONS *options)
```

Comment:
This function seems identical to `msi_verify_nested_signature()` above, except for invoking `pe_verify_pkcs7()` instead of `msi_verify_pkcs7()`.  I'd prefer a single `verify_nested_signature()` function with a callback.